.circleci/config.yml

```
@@ -252,23 +249,20 @@ jobs:
   deploy_pypi:
     machine:
       image: circleci/classic:201711-01
-    working_directory: /tmp/src/smriprep
+    working_directory: /home/circleci/src/smriprep
     steps:
-      - attach_workspace:
-          at: /tmp
+      - checkout
       - run:
-          name: Deploy to PyPi
+          name: Check PyPi preconditions
           command: |
             pyenv global 3.5.2
-            virtualenv venv
-            pip install "setuptools>=27.0" twine future docutils
-            echo "${CIRCLE_TAG}" > smriprep/VERSION
-            echo "include smriprep/VERSION" >> MANIFEST.in
-            python setup.py check -r -s
+            pip install --upgrade "setuptools>=30.3.0" "pip>=18.1" twine docutils
             python setup.py sdist
-            twine upload dist/*
-            cd wrapper && python setup.py sdist
-            twine upload dist/*
+            twine check dist/*.tar.gz
+      - run:
+          name: Deploy to PyPi
+          command: |
+            twine upload dist/*.tar.gz
```

Isn't this necessary?
```suggestion
            pyenv global 3.5.2
            pip install twine
            twine upload dist/*.tar.gz
```

---

No. https://github.com/poldracklab/niworkflows/blob/3ed92ea3b89189c4a86a41d99f6c95eae8829497/.circleci/config.yml#L220-L248